scalameta/semanticdb-scalac-core/src/main/scala/scala/meta/internal/semanticdb/DocumentOps.scala

```diff
@@ -183,10 +183,10 @@ trait DocumentOps { self: DatabaseOps =>
               names(mtree.pos) = symbol
               if (mtree.isDefinition) binders += mtree.pos
 
-              def saveDenotation(): Unit = {
+              def saveDenotation(isDefinition: Boolean): Unit = {
```

Comment:
`val isDefinition = gsym/mtree.isDefinition`?

Reply:
I second this request.